packages/ens-controller/src/EnsController.test.ts

```diff
@@ -50,9 +74,56 @@ describe('EnsController', () => {
           },
         },
       },
+      ensResolutionsByAddress: {},
     });
   });
 
+  it('should clear ensResolutionsByAddress state propery when resetState is called', async () => {
+    const messenger = getMessenger();
+    const controller = new EnsController({
+      messenger,
+      state: {
+        ensResolutionsByAddress: {
+          [address1Checksum]: 'peaksignal.eth',
+        },
+      },
+    });
+
+    expect(controller.state).toStrictEqual({
+      ensResolutionsByAddress: {
+        [address1Checksum]: 'peaksignal.eth',
+      },
+      ensEntries: {},
+    });
+
+    controller.resetState();
+
+    expect(controller.state.ensResolutionsByAddress).toStrictEqual({});
+  });
+
+  it('should clear ensResolutionsByAddress state propery on networkStateChange', async () => {
```

Comment:
@Gudahtt regarding [resetState on network state change](https://github.com/MetaMask/core/pull/1170/files#r1172020116)
I have it tested here

Reply:
Nice, thanks! I missed this addition, this test looks great.